tianshou/utils/net/common.py

```diff
@@ -36,7 +36,10 @@ def __init__(self, layer_num, state_shape, action_shape=0, device='cpu',
     def forward(self, s, state=None, info={}):
         """s -> flatten -> logits"""
         s = to_torch(s, device=self.device, dtype=torch.float32)
-        s = s.flatten(1)
+        if len(s.shape) > 1:
```

Comment:
it is quite ugly x) what is the shape of s ?

Reply:
It is not determined, either [bsz] or [bsz, ?] or [bsz, ?, ?]

Reply:
ok so why not `s.flatten(1, -1)` ?